packages/shared-components/src/components/FileLoader/styles/FileLoaderPreview.js

```diff
@@ -1,5 +1,13 @@
 import styled from 'styled-components';
+import get from 'extensions/themeGet';
 
 export default styled.div`
-  margin: auto auto auto 16px;
+  margin: auto 4px auto 16px;
+  & > * {
+    &:last-child {
+      margin: auto auto auto 2px;
+      color: ${get('colors.primary.alternate')};
+      text-decoration: underline;
```

Comment:
The mockup seems to have bold text here as well?

Reply:
Seems like we should just use an Anchor component here with no click behavior (since the whole input is clickable).